Logger.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+
+public protocol Logger {
+    
+    func logNetworkRequest(request: NSURLRequest) -> Void
```

Comment:
@ashfurrow Agree. I think I can add some comments which will serve as a hint for user what he can implement it his own logger.